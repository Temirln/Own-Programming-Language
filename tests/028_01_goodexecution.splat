program

Num1 : Integer;
Num2 : Integer;

addNumbers(num1 : Integer, num2 : Integer) : Integer is 
	sum : Integer;
begin
	sum := (num1 + num2);
	return sum;
end ;

begin
	Num1 := 17;
	Num2 := 13;
	
	print Num1;
	print" + ";
	print Num2;
	print "= ";
	print addNumbers(Num1, Num2);
	print_line;
end ;