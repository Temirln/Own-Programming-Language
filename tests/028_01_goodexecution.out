17 + 13= 30
